packages/pieces/store/src/lib/actions/store-add-to-list.ts

```diff
@@ -43,8 +43,12 @@ export const storageAddtoList = createAction({
         if (!Array.isArray(items)) {
             throw new Error(`Key ${context.propsValue['key']} is not an array`);
         }
-        if (context.propsValue['ignore_if_exists'] && items.includes(context.propsValue['value'])) {
-            return items;
+        if (context.propsValue['ignore_if_exists'] ) {
```

Comment:
Can you also fix the remove item from the list, to filter everything that is equal to that object?